README.md

```diff
@@ -458,7 +458,9 @@ style.xml
 ```
 
 ## Authorizing Payment
-Some payment methods require the customer to authorize the payment via an authorization URL. This includes the [3-D Secure verification](https://www.omise.co/fraud-protection#3-d-secure), [Internet Banking payment](https://www.omise.co/offsite-payment), [Alipay](https://www.omise.co/alipay), etc. Omise Android SDK provides a built in class to handle the authorization.
+Some payment methods require the customer to authorize the payment via an authorization URL. This includes the [3-D Secure verification](https://www.omise.co/fraud-protection#3-d-secure), [Internet Banking payment](https://www.omise.co/offsite-payment), [Alipay](https://www.omise.co/alipay), etc. Opn Payments Android SDK provides a built in class to handle the authorization.
```

Comment:
@muthuswamyopn can you take the opportunity to replace the omise.co links with their equivalent on docs.opn.ooo (making sure the new link still works)

Reply:
I will make the changes right away today, Dan. Apologies for missing this comment.

Reply:
Updated links.

Should we change email to support@opn.ooo?

Reply:
Updated support email